paimon-format/pom.xml

```diff
@@ -279,7 +282,9 @@ under the License.
                                     <include>org.apache.hive:hive-storage-api</include>
                                     <include>io.airlift:aircompressor</include>
                                     <include>commons-lang:commons-lang</include>
+                                    <include>org.apache.commons:commons-lang3</include>
```

Comment:
relocate for org.apache.commons.lang3?

Reply:
Please check final jar classes.

Reply:
orc.apache.commons.lang3 has already be shaded by 
                                <!-- Relocate Common. -->
                                <relocation>
                                    <pattern>org.apache.commons</pattern>
                                    <shadedPattern>org.apache.paimon.shade.org.apache.commons</shadedPattern>
                                </relocation>

Reply:
![image](https://github.com/apache/incubator-paimon/assets/41894543/d73c016c-7b0f-4bcf-932d-02af23fc93d4)

Reply:
final jar:
![image](https://github.com/apache/incubator-paimon/assets/41894543/aaed4f34-f6b8-40bc-8bff-a08b47628260)